wikimedia/wikipedia-ios
# Notification Center: iPad adjustments

**Phabricator:** https://phabricator.wikimedia.org/T286607

### Notes
* In phab ticket, asked for approval on margins being a bit different than proposed - using `readableContentGuide` instead of requested exact numbers. Olga and Carolyn are comfortable with this.
* iPad cells now smaller. For larger font sizes, cell size will grow.
* Changed the `sourceRect` for the popover so it doesn't block the content. Current behavior was impact of giving a source size of zero.
* As discussed in meeting, we don't have control over the font size or truncation style of pop overs or action sheets, the fourth part of this ticket. If we want to control this, we'd need to switch to another type of control.

### Test Steps
1. Run app, look at Notifications Center and its submenus.
2. Check on both iPad and iPhone, and in various colors

### Screenshots/Videos
Filter margins (before/after)
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 13 17 31](https://user-images.githubusercontent.com/9295855/157982638-5e836ea1-e5b6-4807-81e2-28b68098bf8f.png) ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 13 28 41](https://user-images.githubusercontent.com/9295855/157982826-dbd52812-85be-4165-ab30-0866fd2a8601.png)

Inbox margins (before/after)
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 13 30 18](https://user-images.githubusercontent.com/9295855/157982909-0b3af8e7-f265-4e6a-84de-ffd742e11fde.png) ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 14 02 33](https://user-images.githubusercontent.com/9295855/157982918-90024cc8-3a79-4370-924a-22b16ebeb02d.png)

Updated anchor for popover menu
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 14 33 46](https://user-images.githubusercontent.com/9295855/157983509-6cd5ad68-ed69-4760-aa15-ae36fd5d0b7e.png)

Main screen - margins and cell height (before/after)
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 14 23 33](https://user-images.githubusercontent.com/9295855/157983196-d1273334-ca9a-404c-b366-acf6ecdf886f.png) ![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 14 23 36](https://user-images.githubusercontent.com/9295855/157983209-444c258b-5ce7-440f-8ac4-b3e5e3136d01.png)

Large fonts
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 14 35 45](https://user-images.githubusercontent.com/9295855/157983864-b9424b1c-4951-4d34-b40d-e8146bb451c7.png)
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 14 35 54](https://user-images.githubusercontent.com/9295855/157983868-9d10584d-0c35-4904-b961-
![Simulator Screen Shot - iPad Pro (9 7-inch) - 2022-03-11 at 14 36 16](https://user-images.githubusercontent.com/9295855/157983875-576775ed-c61a-4e67-86ab-417c3b5f4d28.png)
be122cac94ec.png)
